python-sdk/tests/fixtures.py

```diff
@@ -99,7 +99,7 @@ def new_pipeline(
         )
         self.pipelines.append(pipeline)
 
-        with self.pfs.commit(branch=pfs.Branch(repo=repo, name="master")) as commit:
```

Comment:
This one shouldn't be changed. We are specifying the "master" branch purposefully.

Reply:
Done.